tests/providers/aws/lib/allowlist/allowlist_test.py

```diff
@@ -109,7 +109,45 @@ def test_is_allowlisted(self):
                     "Checks": {
                         "check_test": {
                             "Regions": ["us-east-1", "eu-west-1"],
-                            "Resources": ["prowler", "^test"],
+                            "Resources": ["prowler", "^test", "prowler-pro"],
+                        }
+                    }
+                }
+            }
+        }
+
+        assert is_allowlisted(
+            allowlist, AWS_ACCOUNT_NUMBER, "check_test", AWS_REGION, "prowler"
+        )
+
+        assert is_allowlisted(
+            allowlist, AWS_ACCOUNT_NUMBER, "check_test", AWS_REGION, "prowler-test"
+        )
+
+        assert is_allowlisted(
+            allowlist, AWS_ACCOUNT_NUMBER, "check_test", AWS_REGION, "test-prowler"
+        )
+
+        assert is_allowlisted(
+            allowlist, AWS_ACCOUNT_NUMBER, "check_test", AWS_REGION, "prowler-pro-test"
+        )
+
+        assert not (
+            is_allowlisted(
+                allowlist, AWS_ACCOUNT_NUMBER, "check_test", "us-east-2", "test"
+            )
+        )
+
+    def test_is_allowlisted_wildcard(self):
+
+        # Allowlist example
+        allowlist = {
+            "Accounts": {
+                "*": {
+                    "Checks": {
+                        "check_test": {
+                            "Regions": ["us-east-1", "eu-west-1"],
+                            "Resources": [".*"],
```

Comment:
I'd change this to an `*` and the internally include the `.` if the only argument is an `*`. What do you think?

Reply:
Done, great idea!